docs/languages.rst

```diff
@@ -4,11 +4,14 @@
 Language Support
 ================
 
-Rasa NLU can be used to understand any language, but some backends are 
-restricted to specific languages.
+**You can use Rasa NLU to build assistants in any language you want!** The 
+``tensorflow_embedding`` pipeline can be used for **any language** because
+it trains custom word embeddings for your domain. Read more about this
+pipeline in :ref:`choosing_pipeline`.
```

Comment:
also reference `language_support` in here (as we do above)

Reply:
@amn41 Do we really need a reference to language_support here? I think then we will have a reference to the same page.